### ISSUE TITLE
Windows App Certification fails in Supported API test

### ISSUE THREAD

**Issue by tipa** [realm/realm-dotnet#2235]
## Goals
I am trying to submit my UWP app to Microsoft Store. 
No problems with 5.1.2, but after updating to 10.0.1, the app fails during certification
```
API __C_specific_handler in kernel32.dll is not supported for this application type. realm-wrappers.dll calls this API.
```
![image](https://user-images.githubusercontent.com/2863198/107278275-d26c2800-6a55-11eb-9f11-4d648a93e2c5.png)

## Steps to Reproduce
This can also be reproduced by creating a UWP app package in Visual Studio with the checkbox "Generate artifacts to validate the app with the Windows App Certification Kit (WACK)" checked and then running the WACK locally:
https://docs.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps#validate-your-app-package

## Version of Realm and Tooling
- Client SDK Version: 10.0.1

**Comment by nirinchev** [comment]
Hey, thanks for the report - I believe we have identified a fix for this. I'll need to run some more tests locally, but hopefully it will go in this week.

**Comment by tipa** [comment]
Sorry to nag but any timeline when this fix will go live? :) Thanks!

**Comment by nirinchev** [comment]
Sorry, that slipped under the radar - opened https://github.com/realm/realm-dotnet/pull/2258 which backports it to the stable line of releases and will do a release today.

**Comment by tipa** [comment]
Thanks - waiting eagerly :)

**Comment by nirinchev** [comment]
Unfortunately, we've uncovered a regression in the latest Core release - we'll need to get a fix for it before I can cut a .NET one.

**Comment by nirinchev** [comment]
Hey @tipa I apologize for the delay, but we've just released 10.1.1 which should include the fix for UWP certification.
